cypress/support/commands.js

```diff
@@ -75,8 +87,21 @@ Cypress.Commands.add('loginNoCallback', () => {
 
 Cypress.Commands.add('resetTests', () => {
   cy.server();
-  cy.visit('http://localhost:3000');
+  cy.visit('http://127.0.0.1:3000');
   cy.get('#reset-config').click();
-  cy.get('#logout').click();
   cy.window().then(win => win.localStorage.clear());
+  cy.get('[data-cy=use-node-oidc-provider]').click();
+  cy.get('#logout').click();
+});
+
+Cypress.Commands.add('fixCookies', () => {
+  // Temporary fix for https://github.com/cypress-io/cypress/issues/6375
+  if (Cypress.isBrowser('firefox')) {
+    cy.getCookies({ log: false }).then(cookies =>
+      cookies.forEach(cookie => cy.clearCookie(cookie.name, { log: false }))
+    );
+    cy.log('clearCookies');
+  } else {
+    cy.clearCookies();
+  }
```

Comment:
Our tests where failing on firefox, which appears to be a known thing in Cypress that isnt solved yet.